src/GovUk.Education.ExploreEducationStatistics.Content.Model/ContentSection.cs

```diff
@@ -139,9 +139,11 @@ public DataBlock()
 
         public string Name { get; set; }
 
+        public string HighlightName { get; set; }
+        
         public string Source { get; set; }
 
-        public ObservationQueryContext DataBlockRequest { get; set; }
+        public ObservationQueryContext Query { get; set; }
```

Comment:
I was thinking about making the same change. There needs to be something here to maintain backwards comparability with existing DataBlocks cached by the publisher. Alternatively we could take the option of regenerating all the content when this is deployed (which will also help with generating FastTracks for all existing Releases)